libkernelflinger/security.c

```diff
@@ -216,6 +218,123 @@ struct rot_data_t* get_rot_data()
 	return &rot_data;
 }
 
+char* strrpl(CHAR8 *in, const char *src, const char *dst){
+    CHAR8 *p = in;
```

Comment:
Better add a judgement to check the NULL pointer.
if (NULL == in)
    return in;